crowbar_framework/app/models/backup.rb

```diff
@@ -114,10 +117,10 @@ def create_archive
   end
 
   def save_archive
-    self.name = file.original_filename.split(".").first
+    self.name = file.original_filename.remove(".tar.gz")
```

Comment:
We should also validate the extension in the model. Just adapt this https://github.com/MaximilianMeister/crowbar-core/blob/fixup/crowbar-backup/crowbar_framework/app/models/backup.rb#L34 regex.